src/Util/Images.php

```diff
@@ -352,4 +354,15 @@ public static function getBBCodeByUrl(string $photo, string $preview = null, str
 
 		return '[img=' . $photo . ']' . $description . '[/img]';
 	}
+
+	/**
+	 * Get the maximum possible upload size in bytes
+	 *
+	 * @return integer
+	 */
+	public static function getMaxUploadBytes(): int
+	{
+		$upload_size = ini_get('upload_max_filesize') ?: DI::config()->get('system', 'maximagesize');
```

Comment:
Shouldn't the local Friendica config take precedence over the INI value?

Reply:
No. The INI value is the leading value, since this dictates the real maximum size. The config value is only taken, when we don't get the INI value. We accept larger pictures, since we resize them anyway, when they are to big.